public/css/main.css

```diff
@@ -585,18 +586,66 @@ nav .nav-content {
   align-items: center;
   justify-content: center;
 }
-
-.image-banner-1 {
-  background-image: url("http://placeimg.com/940/480/nature");
-}
-
 .banner-text {
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
   font-size: 40px;
   font-weight: bolder;
-  color: #26a9c3;
-  text-transform: uppercase;
+  color: white;
+  text-align: center;
+  font-family: 'Lexend Deca', sans-serif;
+}
+/* Volunteer Application Page Styles*/
+.volunteerPage {
+  width: 100%; 
+  height: 300px;
+}
+.volunteerBannerImg{
+  width: 100%;
+  height: 100%;
+  background-image: url("../img/MVIMG_20190515_201211.jpg");
```

Comment:
This is a new component, but we already have the scrolling image banner component. We should use the existing component instead.